bark_ml/library_wrappers/lib_tf2rl/agents/gail_agent.py

```diff
@@ -38,37 +38,39 @@ def _get_generator(self):
         """Returns instantiated policy -
         parameters from ./examples/example_params/gail_params.json
         """
+        local_params = self._params["ML"]["BehaviorGAILAgent"]
+
         policy = DDPG(
             state_shape=self._environment.observation_space.shape,
             action_dim=self._environment.action_space.high.size,
             max_action=self._environment.action_space.high,
-            lr_actor=self._params["ML"]["BehaviorGAILAgent"]["Generator"]["LearningRateActor", "", 0.001],
-            lr_critic=self._params["ML"]["BehaviorGAILAgent"]["Generator"]["LearningRateCritic", "", 0.001],
-            actor_units=self._params["ML"]["BehaviorGAILAgent"]["Generator"]["ActorFcLayerParams", "", [
+            lr_actor=local_params["Generator"]["LearningRateActor", "", 0.001],
+            lr_critic=local_params["Generator"]["LearningRateCritic", "", 0.001],
+            actor_units=local_params["Generator"]["ActorFcLayerParams", "", [
                 400, 300]],
-            critic_units=self._params["ML"]["BehaviorGAILAgent"]["Generator"]["CriticJointFcLayerParams", "", [
+            critic_units=local_params["Generator"]["CriticJointFcLayerParams", "", [
                 400, 300]],
-            sigma=self._params["ML"]["BehaviorGAILAgent"]["Generator"]["Sigma", "", 0.1],
-            tau=self._params["ML"]["BehaviorGAILAgent"]["Generator"]["Tau", "", 0.005],
-            n_warmup=self._params["ML"]["BehaviorGAILAgent"]["WarmUp", "", 1000],
-            batch_size=self._params["ML"]["BehaviorGAILAgent"]["Generator"]\
-                ["BatchSize", "", 100],
+            sigma=local_params["Generator"]["Sigma", "", 0.1],
+            tau=local_params["Generator"]["Tau", "", 0.005],
+            n_warmup=local_params["WarmUp", "", 1000],
```

Comment:
Can we move the `WarmUp` to the `["Generator"]` part in the params file, as it is only used here once?